tests/query_runner/test_drill.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+import datetime
+from unittest import TestCase
+
+from mock import MagicMock
+
+from redash.query_runner import TYPE_DATETIME, TYPE_FLOAT, TYPE_INTEGER, TYPE_BOOLEAN, TYPE_STRING
+from redash.query_runner.drill import guess_type, convert_type, parse_response, Drill
+
+
+class TestGuessType(TestCase):
```

Comment:
This would go into a separate `tests.query_runner.utils` module once the `guess_type` function has been moved into `redash.query_runner`.